### PR TITLE
Implemented Plan API methods

### PR DESCRIPTION
**Pull request by lukaa12**

### CODE REVIEW

File: kentik_api_library/examples/plans_example.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# pylint: disable=redefined-outer-name
+"""
+Example of using the users API
```
Comment (mateuszmidor) [inline kentik_api_library/examples/plans_example.py:3]: plans API

Comment (lukaa12) [reply kentik_api_library/examples/plans_example.py]: ack

**Review by mateuszmidor — APPROVED** [review]

File: kentik_api_library/kentik_api/public/plan.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from typing import Any, Dict, List, Optional
```
Comment (adrian-szczepanski) [inline kentik_api_library/kentik_api/public/plan.py:1]: Please use @dataclass decorator for all classes in this file (see other modules in .public package for reference)

**Review by adrian-szczepanski — CHANGES_REQUESTED** [review]

File: kentik_api_library/kentik_api/public/plan.py (hunk 2 above)
Comment (lukaa12) [reply kentik_api_library/kentik_api/public/plan.py:1]: ack

**Review by TedatKentik — APPROVED** [review]
Plan APi extension

**Pull request merged**